test/storage/garbage_collector_test.cpp

```diff
@@ -672,4 +672,244 @@ TEST_F(GarbageCollectorTests, InsertUpdate1) {
   }
 }
 
-}  // namespace terrier
+// NOLINTNEXTLINE
```

Comment:
In my opinion, your tests are comprehensive and convincible enough. But it seems that Andy prefers tests without hand-coded answers. For your implementation, I still think it is fine to test in the way you do.

Reply:
@yangdsh I think these hardcoded values are unavoidable in this particular example.

Reply:
Is it better to run a random number of updates here? For example, run n updates and n-1 should be unlinked (except the newest one).